### PR TITLE
feat: toastContainer stub

### PR DESCRIPTION
**Pull request by pozil**
Added a stub for the new [lightning/toastContainer](https://developer.salesforce.com/docs/component-library/bundle/lightning-toast-container/documentation) module.

### CODE REVIEW

File: src/lightning-stubs/toastContainer/toastContainer.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+export const ToastContainer = {
+    instance: jest.fn(() => ({})),
+};
```
Comment (Templarian) [inline src/lightning-stubs/toastContainer/toastContainer.js:9]:
```suggestion
import { LightningElement, api } from 'lwc';

export default class ToastContainer extends LightningElement {
    @api containerPosition;
    @api maxToasts;
    @api toastPosition;
    @api close() {}
}
```

Comment (pozil) [reply src/lightning-stubs/toastContainer/toastContainer.js:9]: @Templarian happy to accept these changes but unless the [lightning/toastContainer documentation](https://developer.salesforce.com/docs/component-library/bundle/lightning-toast-container/documentation) is incorrect we also need a static `instance()` method, no?

Comment (Templarian) [reply src/lightning-stubs/toastContainer/toastContainer.js:9]: @pozil Oh good catch. Quickly copied our internal mock that is missing it also. Updated the code suggestion above to include it.

**Comment by pozil** [comment]
Thanks for the fix. The PR should be good now.

**Review by Templarian — APPROVED** [review]

**Pull request merged**